biojava-structure/src/main/java/org/biojava/nbio/structure/symmetry/core/PermutationGroup.java

```diff
@@ -142,16 +142,16 @@ public int hashCode() {
 
 	@Override
 	public boolean equals(Object obj) {
-	    if (this == obj) {
-		return true;
-	    }
-	    if (obj == null) {
-		return false;
-	    }
-	    if (this.getClass() == obj.getClass()) {
-		return permutations.equals(((PermutationGroup)obj).permutations);
-	    }
-	    return false;
-	}
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (this.getClass() == obj.getClass()) {
+            return getGroupTable().equals(((PermutationGroup) obj).getGroupTable());
+        }
+        return false;
+    }
```

Comment:
I'm not familiar with this class, but it would be good if someone (@sbliven, @lafita, @pwrose  ?)  could check this is alright.